crates/wit-parser/src/resolve.rs

```diff
@@ -3148,6 +3162,7 @@ impl Remap {
         unresolved: &UnresolvedPackage,
         interface_to_package: &HashMap<InterfaceId, (&PackageName, &String, Span)>,
         resolve: &mut Resolve,
+        foreign_interface_stabilities: &HashMap<&Id<Interface>, &Vec<Stability>>,
```

Comment:
For this `InterfaceId` is idiomatically used instead of `Id<Interface>`, and `&[T]` is more idiomatic than `&Vec<T>`